package.json

```diff
@@ -62,7 +63,9 @@
       "git add"
     ],
     "*.scss": [
-      "stylelint --syntax scss"
+      "stylelint --syntax scss --fix",
+      "stylelint --syntax scss",
```

Comment:
stylelint recommends running the command twice, to make sure issues are not missed when running `--fix`